ci/check-copyright.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script makes sure that the checked-in manifest YAML is up-to-date.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+pushd $THIS_DIR/.. > /dev/null
+
+make check-copyright add=true
+diff="$(git status --porcelain `find . -type f -name "*.go"`)"
+
+if [ ! -z "$diff" ]; then
+    echoerr "The copyright of some files is not generated"
+    echoerr "You can regenerate them with 'make check-copyright add=true' and commit the changes"
```

Comment:
I would suggest to rename the target as `add-copyright`, and run `make add-copyright --check-only` to check the file list, so developer can use a shorter CLI to do it.